scripts/install-kustomize.sh

```diff
@@ -62,6 +62,7 @@ echo "$(cat "${KUSTOMIZE_CHECKSUM}")  ${KUSTOMIZE_TARBALL}" | sha256sum -c
 echo "Installing kustomize"
 mkdir -p "${STAGING_DIR}"
 tar -zxvf "${KUSTOMIZE_TARBALL}" -C "${STAGING_DIR}"
+chmod a+x "${STAGING_DIR}/kustomize"
```

Comment:
This can be submitted as a separate change

Reply:
I removed the change and will create a separate PR for it